src/Files.App/Data/Items/DriveItem.cs

```diff
@@ -214,12 +215,16 @@ public object ToolTip
 						Width = 16
 					}
 				};
-				itemDecorator.Click += ItemDecorator_ClickAsync;
+
+				ToolTipService.SetToolTip(itemDecorator, "SideBarEjectDevice/Text".GetLocalizedResource());
```

Comment:
Can you rename the existing string to "Eject"?

Reply:
Are you sure?
If I change it, the resource will be marked as un-translated and it will need to be reviewed again.